src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Overview/Cells/HealthCertifiedPerson/HealthCertifiedPersonCellModel.swift

```diff
@@ -239,3 +242,49 @@ class HealthCertifiedPersonCellModel {
 		}
 	}
 }
+
+extension HealthCertifiedPersonCellModel {
+	enum MaskAndAdmissionStatesConfiguration {
+		/// Show Nothing
+		case maskStatusInvisibleAdmissionStatusInvisible
```

Comment:
Can we rename every where maskStatus to maskState and admissionStatus to admissionState ?

Reply:
👍